packages/astro/test/tailwindcss.test.js

```diff
@@ -62,10 +60,10 @@ describe('Tailwind', () => {
 
 		it('handles Markdown pages', async () => {
 			const html = await fixture.readFile('/markdown-page/index.html');
-			const $ = cheerio.load(html);
-			const bundledCSSHREF = $('link[rel=stylesheet][href^=/assets/]').attr('href');
-			const bundledCSS = await fixture.readFile(bundledCSSHREF.replace(/^\/?/, '/'));
-			expect(bundledCSS, 'includes used component classes').to.match(/\.bg-purple-600{/);
+			const $md = cheerio.load(html);
```

Comment:
Was shadowing the `$` variable shared by all other tests that were loading `/index.html`